awswrangler/catalog/_create.py

```diff
@@ -689,15 +693,15 @@ def create_database(
     ... )
     """
     client_glue: boto3.client = _utils.client(service_name="glue", session=boto3_session)
-    args: Dict[str, str] = {"Name": name}
+    args: Dict[str, Any] = {"Name": name, **database_input_args} if database_input_args else {"Name": name}
     if description is not None:
         args["Description"] = description
 
     try:
         r = client_glue.get_database(**_catalog_id(catalog_id=catalog_id, Name=name))
         if not exist_ok:
             raise exceptions.AlreadyExists(f"Database {name} already exists and <exist_ok> is set to False.")
-        if description and description != r["Database"].get("Description", ""):
+        if args["description"] and description != r["Database"].get("Description", ""):
```

Comment:
Do we now need to check here if either conditions - if database_input_args or description was provided, then update the database?

Reply:
Meaning for parameters outside of `description` we would also update?

Reply:
Yeah, basically to handle the new `database_input_args ` argument

Reply:
@kukushking [How's this?](https://github.com/aws/aws-sdk-pandas/pull/2067/commits/2588c232e55c549d9a81308608daa5bb509db863)

Reply:
This will only handle only one difference because of the `break`, but there may be more changed params. I meant more like this:
```
# If the user provided description or database_input_args parameter
if database_input_args or description:
    # Prepare args
    args: Dict[str, Any] = {"Name": name, **database_input_args} if database_input_args else {"Name": name}
    
    # Update description
    if description is not None:
        args["Description"] = description

     # Update db
    client_glue.update_database(**_catalog_id(catalog_id=catalog_id, Name=name, DatabaseInput=args)) # perform update
```

Reply:
[Description is added to `args` ](https://github.com/aws/aws-sdk-pandas/blob/2064-create-database-support-passing-databaseinput-dict/awswrangler/catalog/_create.py#L698) so any differences between items in `args` and the existing database should trigger a full update.

Reply:
Ah right, thanks. Yeah I think this is fine, but I wouldn't even have a loop to compare the values - I think the fact that user provided `database_input_args ` or `description` is enough to call an update

Reply:
@jaidisido Before we proceed, do you have any thoughts on Anton's suggestion?

Reply:
I think a loop to compare values might actually make sense here because it would avoid unnecessary API calls. As a user I might have a script where Description or DatabaseInputArgs was set at the beginning but I keep running the same script over and over again. Not sure I would want the Update call to be made in that case